### DISCUSSION TITLE
How to show multiple lines in a log?

### DISCUSSION THREAD

**Discussion by zmok8866** [datalust/seq-tickets#1346] [Help] [answered]
Dear

Log show like this:

<L[3] 
      <U2[1] 0  >
      <U4[1] 301  >
      <L[0] 
      >
>

the log can be shown in 5 lines.

How can I do?

**Comment by nblumhardt** [comment, marked as answer]
Hi @zmok8866 - thanks for dropping by!

I'm not sure I understand your question - could you please let me know:

 * Which logging library are you using?
 * What do you see currently? (Screenshot?)

Many thanks,
Nick

**Comment by zmok8866** [comment]
Thank you for your reply!

I used seriLog in my project.

The text  I see currently is:
<L[3]<U2[1] 0 ><U4[1] 301 ><L[0]>

![CurrentImg](https://user-images.githubusercontent.com/30773113/136909816-9a4040f3-27f5-4221-adcd-994c683fe85d.png)


I want to see the text like this:
<L[3]
<U2[1] 0 >
<U4[1] 301 >
<L[0]
>

To see multiple lines in a log.

Waiting for your help, thank you!

**Reply by zmok8866** [reply]
I have try "br" to split lines, but the text is:

![br](https://user-images.githubusercontent.com/30773113/136911082-18e3ede5-be94-4fe3-bc8a-3603d84fc9fa.png)

Which code can split lines?

**Comment by nblumhardt** [comment]
The best way to achieve this is to override Seq's default message styling (which explicitly collapses the newlines/line breaks).

You can do this in _Settings > Instance_, by pasting in a CSS override like:

```css
.rendered-message {
  white-space: pre-wrap;
}
```

Let me know if this helps!

**Reply by goldmode321** [reply]
It works!
